### ISSUE TITLE
Cloudflare Workers compatibility

### ISSUE THREAD

**Issue by thejackshelton** [QwikDev/astro#76]
When using both the `@qwikdev/astro` integration and `@astrojs/cloudflare`, there appears to be an incompatibility issue somewhere that causes the following error with SSR:

```
✘ [ERROR] Uncaught (in promise) Error: NoMatchingRenderer: Unable to render `Counter`.

There is 1 renderer configured in your `astro.config.mjs` file, but it was not able to server-side render `Counter`.
```

reproduction:
https://github.com/vadym-oliinyk/astro-qwik-cloudflare

More context on the issue:
https://discord.com/channels/830184174198718474/1221919048191840428/1221919048191840428

**Comment by thejackshelton** [comment]
Hey friends! Alex has done an awesome job removing the extra esbuild step, which seemed to be causing our issue with workers!

TLDR: you can use cloudflare workers in Qwik with v10 of the adapter.
